src/oss-health/HealthTool.cs

```diff
@@ -17,22 +17,28 @@ namespace Microsoft.CST.OpenSource
 
     public class HealthTool : OSSGadget
     {
+        
+        public HealthTool(ProjectManagerFactory projectManagerFactory) : base(projectManagerFactory)
+        {
+        }
+
         public HealthTool(IHttpClientFactory httpClientFactory) : base(httpClientFactory)
```

Comment:
```suggestion
        public HealthTool(IHttpClientFactory httpClientFactory) : this(new ProjectManagerFactory(httpClientFactory))
```

Reply:
Unless there is something I'm missing this seems like a better pattern.  At the very least we need to choose one to use because there are both ones that case base with httpclientfactory and those that make a projectmanagerfactory based on the httpclient.

Reply:
Yeah I switched all of the OSSGadget CLI tools to this pattern.